CONTRIBUTING.md

```diff
@@ -108,8 +108,11 @@ python -m venv venv
 source venv/bin/activate
 # update the version of pip, setuptools, and wheel
 (venv) pip install -U pip setuptools wheel
-# Install the dependencies in your virtual environment
+# Install all the dependencies in your virtual environment.
+#For the zsh shell, you need to specify quotes around the ".[all]" argument but the code below should work fine for bash shell
 (venv) pip install .[all]
+#Install Graphviz (Also installs the dot program which is needed for graphviz output)
+(venv) sudo apt-get install graphviz
```

Comment:
`apt-get` doesn't have to be run inside the virtualenv (as it is a system-wide install command). Maybe make a separate section about the need for `dot` before line 105, mentioning the `apt-get` install method but linking to the https://graphviz.org/download/  for non-Debian/Ubuntu users

Reply:
Okay.